includes/helper/class-wp-job-manager-helper.php

```diff
@@ -533,46 +551,133 @@ public function licence_error_notices() {
 	}
 
 	/**
-	 * Handles a request on the manage license key screen.
+	 * Handles a request on the manage licence key screen.
+	 *
+	 * @return void
 	 */
 	private function handle_request() {
+		if (
+			empty( $_POST['action'] )
+			|| empty( $_POST['_wpnonce'] )
+			|| ! check_admin_referer( 'wpjm-manage-licence' )
+		) {
+			return;
+		}
+		if ( str_starts_with( sanitize_text_field( wp_unslash( $_POST['action'] ) ), 'bulk_' ) ) {
```

Comment:
I think this method `str_starts_with` is only part of PHP 8.0? And I think we are still expected to support PHP 7.2 based on https://github.com/Automattic/WP-Job-Manager/blob/trunk/wp-job-manager.php#L11

Reply:
It is, but WP also provides a polyfill here: https://github.com/WordPress/WordPress/blob/master/wp-includes/compat.php#L445-L465

In #2392, I bumped the minimum WP version to 6.0, so it should already include that polyfill. :)

Reply:
It is, but WP also provides a polyfill here: https://github.com/WordPress/WordPress/blob/master/wp-includes/compat.php#L445-L465

In #2392, I bumped the minimum WP version to 6.0, so it should already include that polyfill. :)

Reply:
Awesomeeee!